client/packages/invoices/src/Returns/modals/InboundReturn/useDraftInboundReturnLines.ts

```diff
@@ -2,19 +2,25 @@ import React, { useEffect } from 'react';
 import {
   FnUtils,
   InboundReturnInput,
-  InboundReturnLine,
+  GeneratedInboundReturnLineNode,
   InboundReturnLineInput,
   RecordPatch,
 } from '@openmsupply-client/common';
 import { useReturns } from '../../api';
 
 export const useDraftInboundReturnLines = (
-  stockLineIds: string[],
+  outboundReturnLineIds: string[],
   customerId: string
 ) => {
-  const [draftLines, setDraftLines] = React.useState<InboundReturnLine[]>([]);
+  const [draftLines, setDraftLines] = React.useState<
+    GeneratedInboundReturnLineNode[]
+  >([]);
+
+  const data = useReturns.lines.generateInboundReturnLines(
+    outboundReturnLineIds
+  );
+  const lines = data?.nodes;
 
-  const lines = useReturns.lines.inboundReturnLines(stockLineIds);
   const { mutateAsync } = useReturns.document.insertInboundReturn();
 
   useEffect(() => {
```

Comment:
Btw i was never sure why we `map(seed => ({ ...seed }))`, do you know ?

Reply:
Oh haha - I've seen it sometimes to create a new object before mutating, but in this case I think I removed some additional fields and never cleaned this up 😅